### PR TITLE
Change kernel building docker arguments

### PR DESCRIPTION
**Pull request by Sergiu121**

### CODE REVIEW

**Review by valighita — APPROVED** [review]

File: tools/labs/Makefile [hunk 1]
```diff
@@ -36,7 +36,7 @@ docker-docs:
 	cd docker && UID=$(shell id -u) GID=$(shell id -g) docker-compose run docs-build bash -c "cd /linux/tools/labs && make docs"
```
Comment (dbaluta) [inline tools/labs/Makefile:37]: @Sergiu121 please add a small commit message that tells why this change is needed.

**Review by dbaluta — APPROVED** [review]
